tsconfig.json

```diff
@@ -2,14 +2,8 @@
   "extends": "@pocket-tools/tsconfig",
```

Comment:
oh dang i missed this, this should be the workspace package.

Reply:
this is a weird inconsistent linter thing -- otherwise no change. I'm reverting it...

Reply:
@kschelonka Oh this shouldn't extend `"@pocket-tools/tsconfig"` but it should extend `tsconfig/graphql.json` or similar. we can clean this up after. since this wasnt a change you did i just saw it